src/helpers/utils.ts

```diff
@@ -61,3 +65,18 @@ export function getMinMax(data: number[]): IMinMax {
         };
     }, {});
 }
+
+export function disableBucketItemComputeRatio<T extends VisualizationObject.BucketItem>(item: T): T {
+    if (getComputeRatio(item)) {
+        setComputeRatio(item, false);
+    }
+    return item;
+}
+
+export function getComputeRatio(bucketItem: VisualizationObject.BucketItem): boolean {
+    return get(bucketItem, ["measure", "definition", "measureDefinition", "computeRatio"], false);
+}
```

Comment:
Please add unit tests for these functions

Reply:
I think we don't need unit test for these case. Here, don't have logic, it just call a third-party function.